### PR TITLE
Allow `env-setup.py` script to run on GCE as well

### PR DESCRIPTION
**Pull request by jysohn23**
Clean up to use `cloud_tpu_client` library and let users pass in TPU name if running on GCE env (reported by mgorner@).

### CODE REVIEW

File: contrib/scripts/env-setup.py [hunk 1]
```diff
@@ -20,11 +20,18 @@
...
+def update_tpu_runtime(tpu_name, version):
   print(f'Updating TPU runtime to {version.tpu} ...')
-  url = 'http://{tpu_ip}:8475/requestversion/{tpu_version}'.format(
-      tpu_ip=tpu_ip, tpu_version=version.tpu)
-  print('Done updating TPU runtime: {}'.format(requests.post(url)))
+
+  try:
+    import cloud_tpu_client
+  except ImportError:
+    subprocess.call(['pip', 'install', 'cloud-tpu-client'])
```
Comment (dlibenzi) [inline contrib/scripts/env-setup.py:29]: Can this be installed by default?

File: contrib/scripts/env-setup.py [hunk 2]
```diff
@@ -78,6 +74,10 @@ def install_vm(version, apt_packages):
   apt_cmd = ['apt-get', 'install', '-y']
   apt_cmd.extend(apt_packages)
 
+  if not is_root:
```
Comment (dlibenzi) [inline contrib/scripts/env-setup.py:77]: I'd just use `sudo` everywhere and simplify things.

File: contrib/scripts/env-setup.py (hunk 1 above)
Comment (jysohn23) [reply contrib/scripts/env-setup.py:29]: Default as in on Colab and Kaggle runtimes? Colab pushed back previously on any additional packages so I doubt it.

File: contrib/scripts/env-setup.py (hunk 2 above)
Comment (jysohn23) [reply contrib/scripts/env-setup.py:77]: Turns out Kaggle doesn't even have the `sudo` command so I had to case this.

File: contrib/scripts/env-setup.py (hunk 1 above)
Comment (dlibenzi) [reply contrib/scripts/env-setup.py:29]: Covering my eyes to LGTM this 😄 
What about a maybe-sudo here?

Comment (dlibenzi) [reply contrib/scripts/env-setup.py:29]: ... never mind, that's `pip`

**Comment by jysohn23** [comment]
@dlibenzi What do you mean by a maybe sudo for `pip install cloud-tpu-client`?

**Comment by jysohn23** [comment]
> @dlibenzi What do you mean by a maybe sudo for `pip install cloud-tpu-client`?

Just saw your reply.

**Pull request merged**